pages/common/npx.md

```diff
@@ -7,7 +7,7 @@
 
 `npx {{module_name}} {{command_arguments}}`
 
-- In case a package has multiple binaries, specify the package name along with the binary:
+- In case a module has multiple binaries, specify the package name:
```

Comment:
I don't like this example description. what is module vs package?

Reply:
Actually this string it a bit confusing. I made a bit of research about the `npx` jargon and, in the original man page there's:
`npx [options] [-p|--package <pkg>]... <command> [command-arg]...`
Therefore a _package_/_module_ is a JavaScript distributable collection of source code. I will apply some changes keeping this into account.
If the changes will be accepted, I will modify also https://github.com/tldr-pages/tldr/pull/8898